src/ClientMessage.ts

```diff
@@ -1,206 +1,189 @@
-///<reference path="../typings/node/node.d.ts" />
-///<reference path="../typings/long/long.d.ts" />
 /* tslint:disable:no-bitwise */
-/*
- Client Message is the carrier framed data as defined below.
- Any request parameter, response or event data will be carried in the payload.
- 0                   1                   2                   3
- 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
- +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
- |R|                      Frame Length                           |
- +-------------+---------------+---------------------------------+
- |  Version    |B|E|  Flags    |               Type              |
- +-------------+---------------+---------------------------------+
- |                       CorrelationId                           |
- |                                                               |
- +---------------------------------------------------------------+
- |                        PartitionId                            |
- +-----------------------------+---------------------------------+
- |        Data Offset          |                                 |
- +-----------------------------+                                 |
- |                      Message Payload Data                    ...
- |                                                              ...
- */
```

Comment:
Plus, the numbers above seem to mark the field length. If so, then this ASCII diagram is outdated: Correlation ID occupies 64 bits now instead of 32.